WordPress/WordPressHomeWidgetToday/WordPressHomeWidgetToday.swift

```diff
@@ -1,70 +1,52 @@
-//
-//  WordPressHomeWidgetToday.swift
-//  WordPressHomeWidgetToday
-//
-//  Created by Giorgio Ruscigno on 10/15/20.
-//  Copyright © 2020 WordPress. All rights reserved.
-//
-
 import WidgetKit
 import SwiftUI
-import Intents
 
-struct Provider: IntentTimelineProvider {
-    func placeholder(in context: Context) -> SimpleEntry {
-        SimpleEntry(date: Date(), configuration: ConfigurationIntent())
+// TODO - TODAYWIDGET: remove this static model when real data come in.
+let staticModel = TodayWidgetContent(siteTitle: "Around the world with Pam",
+                                   views: 1752,
+                                   visitors: 5000,
+                                   likes: 5000,
+                                   comments: 250)
+
+struct Provider: TimelineProvider {
+
+
+    func placeholder(in context: Context) -> TodayWidgetContent {
+        staticModel
     }
 
-    func getSnapshot(for configuration: ConfigurationIntent, in context: Context, completion: @escaping (SimpleEntry) -> ()) {
-        let entry = SimpleEntry(date: Date(), configuration: configuration)
+    func getSnapshot(in context: Context, completion: @escaping (TodayWidgetContent) -> ()) {
+        let entry = staticModel
```

Comment:
Is there a reason for creating a separate variable here, instead of passing `staticModel` directly in `completion`?

Reply:
Nope. Fixed